src/Resources/Locales/en.json

```diff
@@ -1483,5 +1483,8 @@
             "entityNotFound": "Database not found, cannot execute the query.",
             "forbidden": "Access was denied, you do not have query permissions."
         }
+    },
+    "jobs": {
```

Comment:
it feels like there should be more than one localized string, but maybe you just haven't added any labels or titles to the page yet?